packages/chain-agnostic-permission/src/scope/filter.ts

```diff
@@ -119,47 +118,3 @@ export const getSupportedScopeObjects = (
 
   return filteredScopesObject;
 };
-
-/**
- * Gets all accounts from an array of scopes objects
- * This extracts all account IDs from both required and optional scopes
- * and returns a unique set.
- *
- * @param scopesObjects - The scopes objects to extract accounts from
- * @returns Array of unique account IDs
- */
-export function getCaipAccountIdsFromScopesObjects(
```

Comment:
this guy was just moved into a different file

Reply:
yes